assets/pulpcore-api.prep

```diff
@@ -6,6 +6,9 @@ if -n {
 foreground {
   mkdir -p /etc/pulp/certs
 }
+foreground {
+  dd if=/dev/urandom bs=32 count=1 2>/dev/null | openssl base64 > /etc/pulp/certs/database_fields.symmetric.key
```

Comment:
Please use the same filepath as the installer:
`/etc/pulp/certs/database_fields.symmetric.key`
https://pulp.plan.io/issues/8192#note-18

Reply:
Isn't this the same filepath?